web/static/web/css/header.css

```diff
@@ -1,144 +1,274 @@
+#side-nav > .item {
+    height: calc(var(--font-size) + 2em);
+    margin: calc(var(--nav-space-between) / 2);
+    padding: 0;
+    justify-content: center;
 }
 
-.burger {
-    align-self: center;
-    justify-self: center;
+#nav > .item:hover,
+#side-nav > .item:hover,
+#nav-user-dropdown.active {
+    /* Disables property from Semantic UI */
+    background-color: transparent;
 }
 
-.burger i {
-    color: #FFFFFF;
-    font-size: 30px;
+#nav > .item .bubble-background,
+#side-nav > .item .bubble-background {
+    position: absolute;
+    width: 100%;
+    height: var(--font-size);
+    border-radius: 2em;
+    opacity: 0;
+    transition: width var(--dur) linear, height var(--dur) linear, opacity var(--dur) ease-out;
 }
 
-.nav .ui.inverted.menu .item {
-    color: rgba(255, 255, 255, .7) !important;
+#nav > .item:hover .bubble-background,
+#side-nav > .item:hover .bubble-background {
+    width: calc(100% + var(--bubble-background-extra-width));
+    height: calc(var(--font-size) + 2em);
+    opacity: 1;
 }
 
-.lang-form {
-    display: none;
+#nav > .item > .text,
+#side-nav > .item > .text {
+    z-index: 1;
+    transition: color var(--dur) ease;
 }
 
-@media screen and (max-width: 900px) {
-    .limg.large {
-        display: none;
-    }
+#nav > .item .make_bg_yellow.bubble-background {
+    filter: brightness(90%);
+}
 
-    .limg.small {
-        display: none;
-    }
+#nav > .item .make_bg_turquoise.bubble-background {
+    filter: brightness(90%);
+}
 
-    .limg.medium {
-        display: inline;
+#nav > .item .make_bg_blue.bubble-background,
+#side-nav > .item .make_bg_blue.bubble-background {
+    filter: brightness(130%);
+}
+
+#side-nav {
+    --estimated-max-name-length: 10em;
+
+    margin: 0;
+    justify-self: end;
+    display: grid;
+    grid-template-columns: var(--estimated-max-name-length) calc(var(--logo-padding) + 10px);
+    justify-items: end;
+    font-size: var(--font-size);
+}
+
+#side-nav > .item {
+    margin-right: 0;
+}
+
+#nav-user-dropdown .menu {
+    background-color: rgb(34, 43, 52);
+    border-radius: 7px;
+    /* For some reason doesn't work with `border: 2px rgb(220, 221, 222)` */
+    border: 2px solid rgb(220, 221, 222);
+}
+
+#nav-user-dropdown .menu > .item {
+    /* Fixes weird centering of dropdown items, caused by Semantic UI setting dropdown menu to `display: flex !important`
+       when it's too close to the right window edge */
+    align-self: auto;
+}
+
+#nav-user-dropdown .menu > .item:hover {
+    background-color: rgba(255, 255, 255, .08) !important; /* Overrides Semantic UI */
+}
+
+#side-nav .lang-link {
+    position: absolute;
+    margin: 0;
+    top: 0;
+    right: 0;
+}
+
+#side-nav .lang-link .link {
+    margin: 0;
+    align-self: start;
+}
+
+#side-nav .lang-link .link .text {
+    margin-right: 0.5em;
+}
+
+#side-nav .lang-link .link .globe.icon {
+    margin: 0;
+}
+
+/* Narrower space between elements in navbar */
+@media screen and (max-width: 1100px) {
```

Comment:
Could we use `1200px` instead, so that the change happens at the same time as the change of logo type?

Reply:
True, it looks a bit weird.. Tried removing `--logo-padding: 1em` at [line 170](https://github.com/MAKENTNU/web/pull/168/files#diff-ce9c358c5c2fdf54d88770e2901ab361R170), and it looks a little better. (The logo still gets larger, though - because it's given more space when `--nav-space-between` narrows.) Looks good with your initial suggestion, too. What do you think? Should the sizing be different?

Reply:
Personally, I prefer the my solution. The navbar does get a bit more cramped. However, it feels more natural when reducing the size of my window (e.g if I were to place two windows side by side on my computer screen), due to it being a smoother transition (that is no abrupt change).

Reply:
So replace `1100px` with `1200px` and remove `--logo-padding: 1em`..?

Reply:
Sounds good

Reply:
Nice, no worries :)